src/interval.jl

```diff
@@ -62,6 +62,12 @@ struct Interval{T} <: AbstractInterval{T}
         f, l, inc = if f ≤ l
             f, l, inc
         elseif l ≤ f
+            Base.depwarn(
+                "Constructing an `Interval(x, y, ...)` is where `x > y` is deprecated, " *
+                "use `Interval(y, x, ...)` instead." *
+                sprint(Base.show_backtrace, stacktrace()),
```

Comment:
Normally `depwarn` just shows the first caller but this was inadequate as most calls to this constructor are from an `Interval` constructor.